sentinelsat/__main__.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+from sentinelsat.scripts.cli import cli as main
```

Comment:
Why import as `main` and not leave as `cli`?

Reply:
Just a matter of taste, it seems to me more clear.
But I can change if you want.

Reply:
No, thanks, never mind. 😄